app/policies/application_policy.rb

```diff
@@ -0,0 +1,42 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    raise Pundit::NotAuthorizedError, "must be logged in" unless user
```

Comment:
You don't need that line - it is not related to authorization after all, lack of user means that authentication was not successful - and that something you can handle using `before_action :authenticate_user!` from devise 😉